test-clients/src/main/java/com/hedera/services/bdd/suites/perf/ReadyToRunScheduledXfersLoad.java

```diff
@@ -0,0 +1,176 @@
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.scheduleCreate;
+import static com.hedera.services.bdd.spec.transactions.crypto.HapiCryptoTransfer.tinyBarsFromTo;
+import static com.hedera.services.bdd.spec.utilops.LoadTest.initialBalance;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.inParallel;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.runWithProvider;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+import static com.hedera.services.bdd.suites.perf.PerfUtilOps.stdMgmtOf;
+import static java.util.concurrent.TimeUnit.MINUTES;
+
+public class ReadyToRunScheduledXfersLoad extends HapiApiSuite {
+	private static final Logger log = LogManager.getLogger(ReadyToRunScheduledXfersLoad.class);
+
+	private AtomicInteger numNonDefaultSenders = new AtomicInteger(0);
+	private AtomicInteger numInertReceivers = new AtomicInteger(0);
+
+	private AtomicLong duration = new AtomicLong(Long.MAX_VALUE);
+	private AtomicReference<TimeUnit> unit = new AtomicReference<>(MINUTES);
+	private AtomicInteger maxOpsPerSec = new AtomicInteger(500);
+	private SplittableRandom r = new SplittableRandom();
+
+	public static void main(String... args) {
+		new ReadyToRunScheduledXfersLoad().runSuiteSync();
+	}
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(
+				new HapiApiSpec[] {
+						runReadyToRunXfers(),
+				}
+		);
+	}
+
+	private HapiApiSpec runReadyToRunXfers() {
+		return defaultHapiSpec("RunReadyToRunXfers")
+				.given(
+						stdMgmtOf(duration, unit, maxOpsPerSec)
+				).when(
+						runWithProvider(readyToRunXfersFactory())
+								.lasting(duration::get, unit::get)
+								.maxOpsPerSec(maxOpsPerSec::get)
+				).then(
+						withOpContext((spec, opLog) -> {
+							if (numInertReceivers.get() > 0) {
+								var op = inParallel(IntStream.range(0, numInertReceivers.get())
+										.mapToObj(i -> getAccountBalance(inertReceiver(i)).logged())
+										.toArray(HapiSpecOperation[]::new));
+								CustomSpecAssert.allRunFor(spec, op);
+							}
+						})
+				);
+	}
+
+	static String payingSender(int id) {
+		return "payingSender" + id;
+	}
+
+	static String inertReceiver(int id) {
+		return "inertReceiver" + id;
+	}
+
+	static List<HapiSpecOperation> initializersGiven(int nonDefaultSenders, int inertReceivers) {
+		List<HapiSpecOperation> initializers = new ArrayList<>();
+		for (int i = 0; i < nonDefaultSenders; i++) {
+			initializers.add(
+					cryptoCreate(payingSender(i))
+							.balance(A_HUNDRED_HBARS)
+							.hasRetryPrecheckFrom(NOISY_RETRY_PRECHECKS)
+			);
+		}
+		for (int i = 0; i < inertReceivers; i++) {
+			initializers.add(
+					cryptoCreate(inertReceiver(i))
+							.balance(0L)
+							.hasRetryPrecheckFrom(NOISY_RETRY_PRECHECKS)
+			);
+		}
+
+		for (HapiSpecOperation op : initializers) {
+			if (op instanceof HapiTxnOp) {
+				((HapiTxnOp) op).hasRetryPrecheckFrom(NOISY_RETRY_PRECHECKS);
+			}
+		}
```

Comment:
Seems to me this block is redundant as previous two `for` look already covers the `hasRetryPrecheckFrom` property. Or you meant to add `hasKnownStatusFrom(...)`?

Reply:
Ah, good catch---removed the earlier `hasRetryPrecheckFrom` modifiers. 👍

Reply:
Cool!